app/routes/wallet/credentials.py

```diff
@@ -38,8 +33,9 @@ async def list_credentials(
     Fetch a list of credentials from the wallet
     ---
 
-    The WQL or wallet query language parameter can be used to filter credentials returned from the wallet.
-    The WQL query is a string that can be used to filter records based on the attributes name and value of the record.
+    The WQL (or, wallet query language) parameter can be used to filter credentials returned from the wallet.
+    The `wql` query parameter is a string that can be used to filter records based
+    on the attributes name and value of the record.
```

Comment:
Some duplication here: 
- "The WQL (...) parameter can be used to filter ..."
- "The `wql` query parameter ... can be used to filter ..."

We can probably just have one line, combining the definition of what WQL is, and the fact that `wql` takes a string...

Note: `(or, wallet query language)` can just be `(Wallet Query Langage)`